packages/features/portfolio/src/containers/portfolio-overview/hooks/use-totals-chart.tsx

```diff
@@ -77,7 +77,10 @@ export const useTotalsChart = (baseCurrency: FiatKeys) => {
     [themeVariant, baseCurrency, netWorth]
   );
 
-  if (debts?.isNegative()) {
+  if (
+    debts?.isNegative() &&
+    !barConfig.find((data) => data.dataKey === 'totalAssets')
```

Comment:
Do we have to do this check because of the useMemo above? what if we remove that as well?